include/language/en_us.lang.php

```diff
@@ -3658,3 +3658,12 @@
 $app_strings['MSG_BROWSER_NOTIFICATIONS_ENABLED'] = 'Desktop notifications are now enabled for this web browser.';
 $app_strings['MSG_BROWSER_NOTIFICATIONS_DISABLED'] = 'Desktop notifications are disabled for this web browser. Use your browser preferences to enable them again.';
 $app_strings['MSG_BROWSER_NOTIFICATIONS_UNSUPPORTED'] = 'This browser does not support desktop notifications.';
+
+$app_strings['IMAP_HANDLER_ERROR'] = 'ERROR: {error}; key was: "{key}".';
```

Comment:
Strange... this strings are also additions to PR https://github.com/salesagility/SuiteCRM/pull/6600

Reply:
Yes true. Reason because PR #6600 is a dependency for this PR. So I had to merge it into this branch first. This change will disappearing here when PR #6600 will be merged. (hopefully without any conflict)